Test/Invoke-WhiskeyBuild.Tests.ps1

```diff
@@ -543,3 +588,65 @@ Build:
         $infos | Where-Object { $_ -match 'InformationPreference\ enabled!' } | Should -BeNullOrEmpty
     }
 }
+
+Describe 'Invoke-WhiskeyBuild.when task violates a strict mode rule' {
+    It 'should fail the build' {
+        Init
+        GivenWhiskeyYml @'
+Build:
+- Version:
+    Version: 0.0.0

```

Comment:
Must we always include a `Version` task in whiskey.yml's used for tests? If so, then at least use the more concise format: `- Version: 0.0.0`

Reply:
I was just trying to avoid the stupid Whiskey version warning message. I think I'll make a new PR to remove it. It's dumb at this point.